packages/libs/eda/src/lib/core/components/computations/plugins/correlationAssayMetadata.tsx

```diff
@@ -42,31 +43,16 @@ const cx = makeClassNameHelper('AppStepConfigurationContainer');
  * when those roll out we'll be able to do a little refactoring to make the code a bit nicer.
  */
 
-export type CorrelationAssayMetadataConfig = t.TypeOf<
-  typeof CorrelationAssayMetadataConfig
->;
-
-// eslint-disable-next-line @typescript-eslint/no-redeclare
-export const CorrelationAssayMetadataConfig = t.partial({
-  collectionVariable: VariableCollectionDescriptor,
-  correlationMethod: t.string,
-  prefilterThresholds: FeaturePrefilterThresholds,
-});
-
-const CompleteCorrelationAssayMetadataConfig = partialToCompleteCodec(
-  CorrelationAssayMetadataConfig
-);
-
 export const plugin: ComputationPlugin = {
   configurationComponent: CorrelationAssayMetadataConfiguration,
   configurationDescriptionComponent:
     CorrelationAssayMetadataConfigDescriptionComponent,
   createDefaultConfiguration: () => ({}),
-  isConfigurationComplete: CompleteCorrelationAssayMetadataConfig.is,
+  isConfigurationComplete: CompleteCorrelationConfig.is,
```

Comment:
Will `data2` be `undefined` for this app? If so, not sure this io-ts type will work, since it will enforce that all properties have a value.